src/History/inc/History.h

```diff
@@ -0,0 +1,95 @@
+#ifndef HISTORY_H
+#define HISTORY_H
+
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   History.h
+ * @author Jeff Trent
+ *  @date   Tue, June 22 2021
+ *  @brief  Singleton containing the history
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include <memory>
+#include <vector>
+
+#include "calculator_generated.h"
+#include "Color.h"
+#include "flatbuffers/flatbuffers.h"
+#include "history_generated.h"
+#include "ICalculator.h"
+
+namespace calculator {
+
+/**
+* @brief History interface which exposes to other components
+*
+*/
+
+class History
+{
+    public:
+        /**
+         * @brief delete copy constructors
+         */
+        History(History const&)        = delete;
+        void operator=(History const&) = delete;
+
+        ~History() = default;
+
+        /**
+         * @brief creates a flat buffer object from ICalculator and appends it to calculators_
+         * @param answer [in] float containing the final answer
```

Comment:
if you put the [in] after @param instead of after the param name,, vscode will actually change it's color, pointless but might be nice for readability.